crates/ruff_linter/src/codes.rs

```diff
@@ -1008,6 +1008,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Ruff, "056") => (RuleGroup::Preview, rules::ruff::rules::FalsyDictGetFallback),
         (Ruff, "057") => (RuleGroup::Preview, rules::ruff::rules::UnnecessaryRound),
         (Ruff, "058") => (RuleGroup::Preview, rules::ruff::rules::StarmapZip),
+        (Ruff, "061") => (RuleGroup::Preview, rules::ruff::rules::OperationOnClosedIO),
```

Comment:
Can we use any of the exsiting "holes" in our numbering (assuming they aren't remapped) like 042, 044, 045, 050, 053, 054, 059 instead of introducing new "holes"?